docs/pages/desktop-access/getting-started.mdx

```diff
@@ -22,10 +22,10 @@ This guide requires you to have:
 - An Active Directory domain, configured for LDAPS (Teleport requires an
   encrypted LDAP connection)
 - Access to a Domain Controller
-- An existing Teleport cluster and user, version 8.0 or newer
-  - See [Teleport Getting Started](../getting-started.mdx) if you're new to Teleport
-- A Linux server to run the Teleport Desktop Access service on
-  - You can reuse an existing server running any other Teleport instance
+- An existing Teleport cluster (Teleport Cloud, Open Source, or Enterprise) and user, version 8.0 or newer. 
```

Comment:
Desktop access won't work with cloud until 9.0, so not sure how we want to convey that here.